### DISCUSSION TITLE
List of Chat Commands ( /fly, /grow, /shrink etc)

### DISCUSSION THREAD

**Discussion by ocularvr** [mozilla/hubs#2851] [General Questions & Discussion] [answered]
Hi Everyone,

So far I have only found that the following is applicable to the chat window, we can run these commands:

/shrink
/grow
**Use the "/grow" and "/shrink" commands in the chat box to change the size of your avatar.** 
/fly
**Use the "/fly" command in the chat box to toggle fly mode.** 

Please let me know if you found any more.

Thanks

**Comment by davegoopot** [comment, marked as answer]
From https://hubs.mozilla.com/docs/hubs-features.html  :
```

/leave - Disconnect from the room
/fly - Toggle fly mode
/grow - Increase your avatar's size
/shrink - Decrease your avatar's size
/duck - create a duck object
/debug Toggle physics debug rendering
/vrstats Toggle stats in VR
/scene <scene url> Change the scene (moderators only)
/rename <new name> Rename the room (moderators only)
```

**Reply by robinkwilson** [reply]
Adding this to the above list:
`/audiomode - (experimental) Toggle positional audio.`

Type "/" in the chatbox for a list:
<img width="750" alt="image" src="https://user-images.githubusercontent.com/29560735/90419332-48757080-e06b-11ea-8ce6-07ffbc8551e3.png">
